platform/openide.awt/src/org/openide/awt/HtmlBrowser.java

```diff
@@ -965,6 +965,14 @@ public static URLDisplayer getDefault() {
         public void showURLExternal(URL u) {
             showURL(u);
         }
+        
+        /**
+         * resets the caches used
+         * If the default browser is changed this clears the instantiated objects for the old one
+         */
+        public void clearCache(){
+            
+        }
```

Comment:
This is a new method on a public API interface. As such it will propably require
* an entry in the api changes XML file,
* the module version needs to be increased and
* the javadoc will need an appropriate `@since` tag.

@matthiasblaesing did I miss something?

Reply:
I think you covered everything. I think the only thing missing is `org.netbeans.core` needs to update its dependency on `org.openide.awt` to the newly set version.